### ISSUE TITLE
how to set gradient accumulation parameter while finetuning GPT-3?

### ISSUE THREAD

**Issue by David-Lee-1990** [modelscope/modelscope#92]
I see a config in the code like 
`def cfg_modify_fn(cfg):
    cfg.train.lr_scheduler = {
        'type': 'LambdaLR',
        'lr_lambda': noam_lambda,
        'options': {
            'by_epoch': False
        }
    }
    cfg.train.optimizer = {'type': 'AdamW', 'lr': 1e-5}
    cfg.train.dataloader = {
        'batch_size_per_gpu': 10,
        'workers_per_gpu': 1
    }
    cfg.train.hooks.append({
        'type': 'EvaluationHook',
        'by_epoch': True,
        'interval': 1
    })
    cfg.preprocessor.sequence_length = 512
    cfg.model.checkpoint_model_parallel_size = 1
    return cfg`
But how to set gradient accumulation parameter ? Any detail instruction docs on tuning those params?

**Comment by Firmament-cyou** [comment]
Hi, you can set the gradient accumulation parameters in the configuration.json file, an example is as follows:
```json
{
    "framework": "pytorch",
    "task": "text-generation",
    "preprocessor": {
        "type": "text-gen-jieba-tokenizer"
    },
    "model": {
        "type": "gpt3",
    },
    "pipeline": {
        "type": "gpt3-generation"
    },
    "train": {
        "optimizer": {
            "type": "AdamW",
            "lr": 2e-5,
            "options": {
                "cumulative_iters": 5
            }
        }
    }
}
```
Similarly, you can also use cfg_modify_fn to set the `cumulative_iters` parameter, which is actually more concise:
```python
def cfg_modify_fn(cfg):
    cfg.train.optimizer.options.cumulative_iters = 5
    return cfg
```

**Comment by David-Lee-1990** [comment]
@Firmament-cyou Thanks for your reply. 

By th way, I am training GPT3-1.3B on my own data which is about 0.2B samples.  I use 8*V100-32G, the log shows the whole process need 188 days. how can i accelerate the training speed?

![image](https://user-images.githubusercontent.com/29653610/218623630-f313e1bd-fe5a-4db7-a1dc-ee8534231682.png)
